src/unwinder.rs

```diff
@@ -575,6 +592,13 @@ enum ModuleUnwindDataInternal<D: Deref<Target = [u8]>> {
         debug_frame: Arc<D>,
         base_addresses: crate::dwarf::BaseAddresses,
     },
+    /// Used with PE binaries (Windows).
+    PeUnwindInfo {
+        pdata: D,
+        rdata: Option<(Range<u32>, D)>,
+        xdata: Option<(Range<u32>, D)>,
+        text: Option<(Range<u32>, D)>,
```

Comment:
It would be nice to have a comment somewhere close by about the range being an RVA range. Or maybe it's worth having a `DataAtRvaRange<D>` struct.